app/Providers/SettingsServiceProvider.php

```diff
@@ -150,6 +150,8 @@ public function boot()
         // Set the monetary locale to the configured locale to make helper::parseFloat work.
         setlocale(LC_MONETARY, config('app.locale'));
         setlocale(LC_NUMERIC, config('app.locale'));
+
```

Comment:
I'd rather not have these spaces, but that's absolutely no hill I would remotely die on.